### PR TITLE
Add quest for powerpoles material type

### PR DESCRIPTION
**Pull request by ENT8R**
This PR adds a new quest to determine the material type of a power pole. This quest was requested in #187.
It looks like this:

Yes, I know, this quest doesn't have an icon yet, because I don't know how to implement it... But maybe the ["high voltage" emoji from twemoji](https://commons.wikimedia.org/wiki/File:Twemoji2_26a1.svg) is a good icon...

### CODE REVIEW

File: app/src/main/java/de/westnordost/streetcomplete/quests/powerpoles_material/AddPowerPolesMaterial.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+{
+	@Inject public AddPowerPolesMaterial(OverpassMapDataDao overpassServer)
+	{
+		super(overpassServer);
+	}
+
+	@Override
+	protected String getTagFilters()
+	{
+		return "nodes, ways with power=pole and !material";
```
Comment (westnordost) [inline app/src/main/java/de/westnordost/streetcomplete/quests/powerpoles_material/AddPowerPolesMaterial.java]: Poles are nodes, remove `ways`, please

File: app/src/main/java/de/westnordost/streetcomplete/quests/powerpoles_material/AddPowerPolesMaterialForm.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import android.view.ViewGroup;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.quests.ImageListQuestAnswerFragment;
+
+public class AddPowerPolesMaterialForm extends ImageListQuestAnswerFragment
+{
+	private final OsmItem[] TYPES = new OsmItem[] {
+			new OsmItem("wood", R.drawable.power_pole_wood, R.string.quest_powerPolesMaterial_wood),
+			new OsmItem("metal", R.drawable.power_pole_metal, R.string.quest_powerPolesMaterial_metal),
```
Comment (westnordost) [inline app/src/main/java/de/westnordost/streetcomplete/quests/powerpoles_material/AddPowerPolesMaterialForm.java]: It's more commonly tagged as `steel`, not `metal`. Which is more correct, IMO.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -381,4 +381,8 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="produce_bell_peppers">Bell peppers</string>
     <string name="produce_brazil_nuts">Brazil nuts</string>
     <string name="produce_tung_nuts">Tung nuts</string>
+    <string name="quest_powerPolesMaterial_title">What kind of material is this?</string>
+    <string name="quest_powerPolesMaterial_wood">Wood</string>
+    <string name="quest_powerPolesMaterial_metal">Metal</string>
```
Comment (westnordost) [inline app/src/main/res/values/strings.xml]: See comment above.

**Comment by westnordost** [comment]
Hmm, actually, the picture for the metal material doesn't show a pole, but a tower. So it should be exchanged with one that shows a pole.

**Comment by ENT8R** [comment]
I changed the image, strings and query.
The image for "steel" is now:


Another question:
Should I change the question title to something like "What is the material of this power pole?"

**Comment by westnordost** [comment]
Oh yeah, that'd make sense.

**Comment by ENT8R** [comment]
The quest title is now changed. From my point of view, everything is finished now...

**Comment by westnordost** [comment]
Ok, so just the icon. I'll do this in the next days.

**Comment by ENT8R** [comment]
Perfect!

**Comment by Etua** [comment]
I can try to craft one, but for now I can't give the exact date (like with car wash) so I'll try to send it in a few days or I will tell if I were unable to.

**Comment by westnordost** [comment]
There is an icon now

**Comment by gustavecha** [comment]
Hi everyone,
Really big thank you for this, it looks really good and accurate :)

Can I help you for translating in French eventually ?

**Comment by ENT8R** [comment]
@gustavecha You can help to translate the app at [POEditor](https://poeditor.com/join/project/IE4GC127Ki)

File: app/src/main/java/de/westnordost/streetcomplete/quests/powerpoles_material/AddPowerPolesMaterialForm.java (hunk 2 above)
Comment (matkoniecz) [reply app/src/main/java/de/westnordost/streetcomplete/quests/powerpoles_material/AddPowerPolesMaterialForm.java]: And as https://poeditor.com/projects/po_edit?id=97843&term=67125651&id_language=127#showComments67125651 points out

> Probably misleading that it says metal there at all but I've checked the code and this option does correspond to the tag 'steel', not 'metal', being added.

**Pull request merged**